### PR TITLE
Plant Refactor: It came from the Conflicts Hell

### PR DESCRIPTION
**Pull request by Core0verload**
This PR splits `seeds.dm` and `growninedible.dm` into subfiles, just like #15880 splitted `grown.dm`. It also adds the most common variables to the base seed type. All changes are aimed at reducing amount of repeated code in plants code and making it more condensed. If everything is done right, players shouldn't notice any changes. 

I added a proc for checking seeds for broken in-tray icons, potentially useful as unit test. It found that 3 plants had broken icons all the time: mimana, carpet and glowcap. I didn't fixed it yet, and I'm not sure about mimana - maybe invisible mime banana is a feature?

**It changes a fuckton of seeds paths** because:
1. The coder who added all the seeds has clearly forgot about such things as "inheritance".
2. We are not Department of Redundancy Department and having seeds with paths like `/obj/item/seeds/bananaseeds` isn't our specialty.

All files, including map files, are updated for new seeds paths. Yes, I changed ALL MAP FILES in a single PR, because I had to. **If this PR will not be quickmerged, it will forever remain in the sad state of perpetual conflicts.**

This PR is a part of update line that will greatly reduce amount of clutter in plants code/sprites and, at the last stage, will add bio-engineering - system similar to Bay's xenobotany, except aimed at non-autistic playerbase. The next PR in the line will touch plant icons and plant icon generation procs.

### CODE REVIEW

**Comment by Jordie0608** [comment]
For the conflicts you may be interested in https://github.com/tgstation/-tg-station/blob/master/code/__PATH_COMPATIBILITY/path_compatibility.dm

**Comment by Core0verload** [comment]
Yes, I know about this, but it feels like a dirty hack, even dirtier than quickmerge-begging.

**Comment by Fox-McCloud** [comment]
Who thought having 100 different maps to maintain, at once, when only 2 actually matter, was a good idea?

**Comment by Cheridan** [comment]
I'll go ahead and merge this. It's just a refactor so there isn't anything to argue about here.

File: code/modules/hydroponics/grown/nettle.dm [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+	origin_tech = "combat=3"
+
+/obj/item/weapon/grown/nettle/death/add_juice()
+	..()
+	reagents.add_reagent("facid", round((potency / 2), 1))
+	force = round((5 + potency / 2.5), 1)
+
+/obj/item/weapon/grown/nettle/death/pickup(mob/living/carbon/user)
+	..()
+	if(..())
```
Comment (phil235) [inline code/modules/hydroponics/grown/nettle.dm]: Calling ..() twice? That's a typo.

**Pull request merged**